framework_tvb/tvb/tests/framework/core/services/project_service_test.py

```diff
@@ -300,7 +300,7 @@ def test_project_disk_size(self):
 
             ratio = float(actual_disk_size) / project.disk_size
             msg = "Real disk usage: %s The one recorded in the db : %s" % (actual_disk_size, project.disk_size)
-            assert ratio < 1.4, msg
+            assert ratio < 1.7, msg
```

Comment:
I do not think this change is healthy. We should rather find why the disk size is so much different. It should be very close to 1.0 ratio

Reply:
Fixed in PR #252. At first, I thought all files on the disk were taken into consideration when computing the disk usage of a project. And since we moved from Operation.xml to ViewModel.h5, the disk usage would have increased.
But now I see that only the H5 files were taken into consideration before, so I made a fix to exclude the view models.

Reply:
Maybe before the operation.xml files were considered irrelevant, or it was a bug. I think VMs should be considered in the calculation, as they can be many and sometimes large.